frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -24,6 +24,7 @@ function ReaderCoptListener:onReadSettings(config)
     -- crengine top status bar can only show author and title together
     self.title = G_reader_settings:readSetting("cre_header_title") or 1
     self.clock = G_reader_settings:readSetting("cre_header_clock") or 1
+    self.clock_auto_refresh = G_reader_settings:readSetting("cre_header_clock_auto_refresh") or 1
```

Comment:
Does this need a separate setting? There's already an auto-refresh clock setting.

Reply:
I think the refresh of the header is different from the footer.
2.) If the clock in the footer is disabled, the refresh cannot be toggled. (Maybe the logic behind that could be improved: First you have to enable "Current time" in bottom bar, then you can toggle the autorefresh in settings). All the other switches in the bottom menu bar are not connected to the top bar, so mixing seems unlogically for me.
2.) The header gets refreshed **every minute**, if the **time** changes **or** (if time is disabled) the **battery** status changes (if battery status is enabled). So you are right -> a better name would be "cre_header_auto_refresh" as it is not only connected to time.